service/awsconfig/v10/resource/s3bucket/create.go

```diff
@@ -68,3 +86,41 @@ func (r *Resource) newCreateChange(ctx context.Context, obj, currentState, desir
 
 	return BucketState{}, nil
 }
+
+func (r *Resource) createAccessLogBucket(ctx context.Context, bucketInput BucketState) error {
+	_, err := r.clients.S3.CreateBucket(&s3.CreateBucketInput{
+		Bucket: aws.String(bucketInput.Name + "-logs"),
+	})
+	if IsBucketAlreadyExists(err) || IsBucketAlreadyOwnedByYou(err) {
+		// Fall through.
+		return nil
+	}
+	if err != nil {
+		return err
+	}
+
+	_, err = r.clients.S3.PutBucketAcl(&s3.PutBucketAclInput{
+		Bucket:       aws.String(bucketInput.Name + "-logs"),
```

Comment:
We should use a key function for the bucket name and a constant for the S3 URIs.